notebooks/Emukit-Bayesian-optimization-benchmark.ipynb

```diff
@@ -0,0 +1,198 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Emukit Bayesian optimization Benchmark\n",
+    "\n",
+    "This notebook compares multiple Bayesian optimization methods against each other, using the Branin test function."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import emukit\n",
+    "import numpy as np"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Set up test function\n",
+    "\n",
+    "We will use the Branin function which is already included in Emukit, both the function and the appropriate input domain are ready made for us."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from emukit.test_functions.branin import branin_function\n",
+    "branin_fcn, parameter_space = branin_function()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Set up methods to test\n",
+    "\n",
+    "We will compare Bayesian optimization using different models. All the methods will collect points one at a time in a sequential fashion and use the expected improvement acquisition function. The models we test are:\n",
+    "- A Gaussian process with Matern52 covariance function\n",
+    "- Random forest using the pyrfr package\n",
+    "- Bayesian neural network using the Bohamiann package\n",
```

Comment:
Did you use Bayes nets after all?

Reply:
Oops no, it took too long to train, I'll remove this line